posts/2024/2024_08_22_inigo_final_report.rst

```diff
@@ -0,0 +1,309 @@
+
+* I opened a PR that got merged into the main DIPY repository, which was a `nomenclature issue with spherical harmonics <https://github.com/dipy/dipy/issues/2970>`_. It took some time to agree on how to solve it, but it was very nice to see that the community was open to discussing the issue and to find a solution that was good for everyone. This was the contribution that gave me access to GSoC with DIPY, and it was a very nice start to the journey. Link to the PR: https://github.com/dipy/dipy/pull/3086
+* I reviewed the code of my fellow GSoC students `Kaustav <https://github.com/deka27>`_, `Wachiou <https://github.com/WassCodeur>`_ and `Robin <https://github.com/robinroy03>`_. It felt very good to understand their projects and to be able to help them with their work, which is completely different from my project. I also reviewed their blogs and participated in the reviews I got from them. It was very pleasant to see how engaging the community is, and how everyone is willing to help each other.
+* Lastly, I opened an `issue in the dipy.org repository <https://github.com/dipy/dipy.org/issues/40>`_ that got solved thanks to a contribution from my mentor `Serge Koudoro <https://github.com/skoudoro>`_.
+
+|future-work-title|
+-------------------
+
+I plan to continue with this project until I arrive to a satisfactory solution to generate synthetic human tractograms with tuneable age and neurodegeneration status properties. This project has not only been a great learning experience, but it has also been a great challenge and a great addition to my PhD research, so I will for sure continue until I succeed.
+
+To complete the work, I will need to:
+
+* Complete the objectives in progress.
+* Train and validate the models with human tractograms.
+* Integrate the models into DIPY.
+
+|pull-requests-title|
+---------------------
+
+Because the work is still in progress, no integration of the final version was done yet. However, a draft Pull Request was created to show the progress of the work in the main DIPY repository. For now, it contains the first validated AE, together with tests for the model class. `LINK TO THE PR <https://github.com/dipy/dipy/pull/3328>`_
+
+|conclusions-title|
+-------------------
+
+I am super grateful for the opportunity to participate in the Google Summer of Code program with DIPY. This journey has been amazing as learning and research experience. I am proud of the progress I have made and what I have achieved. Nevertheless, this is just the beginning of my journey with this project.
+
+I would like to thank again my mentors, Jon Haitz, Jong Sung and Serge, their time, patience, attention, support, and expertise throughout the program. They have been a key part of my success. On the other hand, I would like to thank my fellow GSoC students, Kaustav, Wachiou, and Robin, for their support and collaboration. It has been a pleasure to work with them and to learn from them.
```

Comment:
It was awesome to learn from you too!

Reply:
Thank you @itellaetxe from everything

Reply:
@itellaetxe It would be cool if you could quote them with their Github profiles. 😇

Reply:
Thank you @robinroy03, @WassCodeur!
I will link everyone's names to their GitHub profiles, okay. Thanks